runtime/executor/src/main/java/org/apache/nemo/runtime/executor/data/BlockManagerWorker.java

```diff
@@ -515,7 +527,7 @@ private BlockStore getBlockStore(final DataStoreProperty.Value blockStore) {
       case LOCAL_FILE:
         return DataStoreProperty.Value.LocalFileStore;
       case REMOTE_FILE:
-        return DataStoreProperty.Value.GlusterFileStore;
```

Comment:
Actually, I got stuck here without a solution: Converting the ControlMessage REMOTE_FILE to either CrailFileStore or GlusterFileStore. For now, I just substituted GlusterFileStore to CrailFileStore, and let GlusterFileStore be converted to LocalFileStore, which absolutely isn't the right way :( Any suggestions?